src/features/home/courses/courses.scss

```diff
@@ -47,71 +47,52 @@
         }
 
         display: flex;
```

Comment:
I thought we decided to use CSS-modules. Or will we still thinking about to use Tailwind CSS?

Reply:
As far as I'm concerned, the final decision hasn't been made yet.

Reply:
I suppose we have to do our best for now (specially it takes only 1-2 minutes of your time), because at the moment we don't know the structure of our project in the future